src/components/Create/components/RecallCard/hooks/FundingTarget.ts

```diff
@@ -0,0 +1,38 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { useAppSelector } from 'hooks/AppSelector'
+import { V2V3CurrencyOption } from 'models/v2v3/currencyOption'
+import { formatCurrencyAmount } from 'utils/formatCurrencyAmount'
+import { MAX_DISTRIBUTION_LIMIT } from 'utils/v2v3/math'
+
+function formatFundingTarget({
+  distributionLimit,
+  distributionLimitCurrency = '1',
```

Comment:
**suggestion** I beleive theres a constant for this that we can use instead of hardcoding.